js/data/siteHacks.js

```diff
@@ -22,13 +22,17 @@ const emptyDataURI = {
 }
 
 /**
- * Holds an array of [Primary URL, subresource URL] to allow 3rd party cookies.
+ * Holds an map of {Primary URL: subresource URL} to allow 3rd party cookies.
  * Subresource URL can be '*' or undefined to indicate all.
  */
-module.exports.cookieExceptions = [
-  ['https://inbox.google.com', 'https://hangouts.google.com'],
-  ['https://mail.google.com', 'https://hangouts.google.com']
-]
+module.exports.cookieExceptions = {
```

Comment:
this will probably eventually need to be a map of domains to list, like `'https://inbox.google.com': [...]` because we will need multiple cookie exception origins for the same top-level origin